src/platform/platform_linux.cpp

```diff
@@ -76,7 +57,7 @@ bool mp::platform::is_backend_supported(const QString& backend)
 
 mp::VirtualMachineFactory::UPtr mp::platform::vm_backend(const mp::Path& data_dir)
 {
-    const auto& driver = get_driver_str();
+    const auto& driver = utils::get_driver_str();
```

Comment:
Definitely not going to block on it, but I prefer putting the `mp` namespace before any `utils` calls.

Reply:
Why?

Reply:
Just seems clearer to me where is comes from.  But like I said, I'm not going to block or anything if you don't want to change it.  It's just a preference of mine.

Reply:
OK, I don't quite see it that way. We're in the multipass namespace, it is only natural to assume it (and that is what the compiler does too - I believe it is an msvc bug that it does not do that for parameters and trailing returns). If we were to prefix every multipass function call with a `mp::` namespace the code would be much noisier to read.

Reply:
I'm of @ricab 's disposition here, but I'm also ok with defining "mpu = mp::utils" and using that.

Reply:
Sure, that's fine.  I just remember when we were evaluating a 3rd party library (I can't remember what it was now), it also had a `utils` namespace, so there was a collision.  I found that `utils` is a fairly common namespace for other libraries, and so I went with the `mp::utils` convention.  But we aren't using anything like that now, so meh.  This was only meant as a suggestion since it's like that in most places in the code and didn't mean for it turn into a longer conversation :smiley: